src/tribler-core/tribler_core/modules/torrent_checker/torrent_checker.py

```diff
@@ -209,6 +210,8 @@ def update_torrents_checked(self, new_result):
         new_result_tuple = (new_result['infohash'], new_result['seeders'],
                             new_result['leechers'], new_result['last_check'])
         self.torrents_checked.add(new_result_tuple)
+        if new_result['seeders'] > 0:
+            self.torrents_checked.add(new_result_tuple)
```

Comment:
You are now adding the `new_result_tuple` twice. Is this intended?

Reply:
Good point! Thank you for pointing it out